platforms/python/versionsToBuild.txt

```diff
@@ -11,8 +11,10 @@
 3.9.0, E3FF2839C048B25C084DEBE9B26995E310250568,
 3.9.1rc1, E3FF2839C048B25C084DEBE9B26995E310250568,
 3.9.7, E3FF2839C048B25C084DEBE9B26995E310250568,
+3.9.12, E3FF2839C048B25C084DEBE9B26995E310250568,
 
-3.10.0a2, 64E628F8D684696D,
+3.10.0a2, A035C8C19219BA821ECEA86B64E628F8D684696D,
 3.10.0, A035C8C19219BA821ECEA86B64E628F8D684696D,
+3.10.4, A035C8C19219BA821ECEA86B64E628F8D684696D,
 
```

Comment:
For updating python versions, are we missing to update the following files ? -
1. build/__pythonVersions.sh
2. src/BuildScriptGenerator/PythonVersions.cs